### PR TITLE
Assignment5

### PR DESCRIPTION
**Pull request by AyanF**
Serialization completed, Deserialization working for both models

### CODE REVIEW

File: ObjectStreaming/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (yashwantdhakad) [inline ObjectStreaming/.project:1]: Can you please all these extra files into .gitignore?

File: ObjectStreaming/src/DeserializationTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          scanner.close();
+          
+        }
+        
+        catch(IOException ioe){
+            
+            ioe.printStackTrace();
+            
+        }
+
```
Comment (yashwantdhakad) [inline ObjectStreaming/src/DeserializationTest.java:34]: Can you please add proper indentation?

**Review by yashwantdhakad — APPROVED** [review]
Approved with minor suggestions

File: ObjectStreaming/.project (hunk 1 above)
Comment (AyanF) [reply ObjectStreaming/.project:1]: sure sir

File: ObjectStreaming/src/DeserializationTest.java (hunk 2 above)
Comment (AyanF) [reply ObjectStreaming/src/DeserializationTest.java:34]: Done

**Pull request merged**